src/Form/Type/Security/ForgotPasswordType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Smart\AuthenticationBundle\Form\Type\Security;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @author Nicolas Bastien <nicolas.bastien@smartbooster.io>
+ */
+class ForgotPasswordType extends AbstractType
+{
+    /**
+     * @inheritdoc
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('email', EmailType::class, [
+                'label' => 'forgot_password.label_email',
```

Comment:
You named it `forgot_password.label_username` on the security.en.xliff

Reply:
Ok I remove this cause I only use place holder